docs/en/sql-reference/aggregate-functions/reference/quantiletimingweighted.md

```diff
@@ -68,15 +68,15 @@ Input table:
 Query:
 
 ``` sql
-SELECT quantileTimingWeighted(response_time, weight) FROM t
+SELECT quantilesTimingWeighted(response_time, weight) FROM t
 ```
 
 Result:
 
 ``` text
-┌─quantileTimingWeighted(response_time, weight)─┐
-│                                           112 │
-└───────────────────────────────────────────────┘
+┌─quantilesTimingWeighted(response_time, weight)─┐
```

Comment:
quantile**s**TimingWeighted should return array, but there are scalar value in the example. Maybe we need to add information for both functions to this doc?

Reply:
>Maybe we need to add information for both functions to this doc?

You are right, thanks, will update the patch